JavaScript/2-benchmark.js

```diff
@@ -3,15 +3,16 @@
 const benchmark = {};
 module.exports = benchmark;
 
-benchmark.do = (count, retry, tests) => {
+benchmark.do = (count, retry, tests, namesOfTests) => {
 
   let k;
   for (k = 0; k < retry; k++) {
-    tests.map(test);
+    let counter = 0; // Order number of function that will be call
```

Comment:
Aside from that, consider rewording the comment itself. First of all, please fix a grammar mistake (s/call/called). Secondly, "order number" is quite a harsh word-to-word translation from Russian. In English it sounds like someone has ordered to buy a function and this variable contains the ID of their order :) "Index" would be fine here.